WasteWar/Assets/Scripts/Camera Controls/Move.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
+
+public class Move : MonoBehaviour
+{
+    [SerializeField]
+    private Terrain terrain;
+    [SerializeField]
+    private Transform cam;
+
+    private Vector3 terrainSize;
+
+    void TryToMoveForward()
```

Comment:
you're duplicated code with CameraConstants.SCREEN_BORDER
void TryToMoveForward()
{
if(keyboardpress || MouseAtBorder())
MoveForward()
}